src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs

```diff
@@ -60,14 +60,14 @@ public static List<SearchResult> MatchUpExistingCompletedStandards(this List<Sea
                 logger.LogInformation("MatchUpExistingCompletedStandards After GetCertificateLogsFor");
                 var submittedLogEntry = certificateLogs.FirstOrDefault(l => l.Status == CertificateStatus.Submitted);
                 var createdLogEntry = certificateLogs.FirstOrDefault(l => l.Status == CertificateStatus.Draft);
-                if (request.IsPrivatelyFunded)
+                var toBeApprovedLogEntry = certificateLogs.FirstOrDefault(l => l.Status == CertificateStatus.ToBeApproved);
+
+                if(toBeApprovedLogEntry != null)
                 {
-                    var toBeApprovedLogEntry = certificateLogs.FirstOrDefault(l => l.Status == CertificateStatus.ToBeApproved);
-                    if (toBeApprovedLogEntry == null)
-                        continue;
+                    // To Be Approved only exists for private funded certificates
```

Comment:
What's going on here?

Reply:
```if (request.IsPrivatelyFunded)
                {
                    var toBeApprovedLogEntry = certificateLogs.FirstOrDefault(l => l.Status == CertificateStatus.ToBeApproved);
                    if (toBeApprovedLogEntry == null)
                        continue;
                    submittedLogEntry = toBeApprovedLogEntry;
                }```
                
This was the original logic. Privately Funded is no longer part of a search result, so wasn't sure how to re-work the logic. If it was privately funded and the to be approved existed it went over submitted, is it best just to take it out do you think?